app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Update Post</h1>
+<%= render 'form'%>
+<%= link_to 'Cancel and Return to posts listing', posts_path%>
```

Comment:
Please add this in main file so that no need to write the same line multiple times in multiple files